src/store/modules/user/getters.ts

```diff
@@ -19,7 +19,8 @@ const getters: GetterTree <UserState, RootState> = {
         return state.selectedBrand
     },
     getInstanceUrl (state) {
-        return state.instanceUrl;
+        const baseUrl = process.env.VUE_APP_BASE_URL;
+        return baseUrl ? baseUrl : state.instanceUrl;
```

Comment:
Remove the baseUrl check from the api/index.ts file, as now we will be always sending the baseUrl from the getter itself.